redirects.csv

```diff
@@ -74,7 +74,22 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /series/* autoPlay=:autoPlay,/media/series/:splat?autoPlay=:autoPlay,302!
 /series/* sound=:sound,/media/series/:splat?sound=:sound,302!
 /series/*,/media/series/:splat,302!
-/articles/*,/media/articles/:splat,302!
+/media/articles/filters//page/10,https://${env:CRDS_MEDIA_ENDPOINT}/articles/page/10,301!
```

Comment:
The double slashes here are intentional. Apparently somewhere in the system someone has created an href with the double slashes and the organization wants us to account for that and redirect.